src/form/en_ivr.jl

```diff
@@ -0,0 +1,1763 @@
+		pm::AbstractExplicitNeutralIVRModel,
+		nw::Int,
+		f_idx::Tuple{Int,Int,Int},
+		f_connections::Vector{Int},
+		rate_a::Vector{<:Real}
+	)
+
+For IVR models with explicit neutrals,
+imposes a bound on the from-side line power magnitude.
+"""
+function constraint_mc_thermal_limit_from(pm::AbstractExplicitNeutralIVRModel, nw::Int, f_idx::Tuple{Int,Int,Int}, f_connections::Vector{Int}, rate_a::Vector{<:Real})
+    vr_fr = [var(pm, nw, :vr, f_idx[1])[t] for t in f_connections]
+    vi_fr = [var(pm, nw, :vi, f_idx[1])[t] for t in f_connections]
+    cr_fr = var(pm, nw, :cr, f_idx)
+    ci_fr = var(pm, nw, :ci, f_idx)
+
+    for idx in 1:length(rate_a)
+        if rate_a[idx]<Inf
+            psw_fr_idx = JuMP.@NLexpression(pm.model,  vr_fr[idx]*cr_fr[idx] + vi_fr[idx]*ci_fr[idx])
+            qsw_fr_idx = JuMP.@NLexpression(pm.model, -vr_fr[idx]*ci_fr[idx] + vi_fr[idx]*cr_fr[idx])
+
+            JuMP.@NLconstraint(pm.model, psw_fr_idx^2 + qsw_fr_idx^2 <= rate_a[idx]^2)
+        end
+    end
+end
+
+
+"""
+	function constraint_mc_thermal_limit_to(
+		pm::AbstractExplicitNeutralIVRModel,
+		nw::Int,
+		t_idx::Tuple{Int,Int,Int},
+		t_connections::Vector{Int},
+		rate_a::Vector{<:Real}
+	)
+
+For IVR models with explicit neutrals,
+imposes a bound on the to-side line power magnitude.
+"""
+function constraint_mc_thermal_limit_to(pm::AbstractExplicitNeutralIVRModel, nw::Int, t_idx::Tuple{Int,Int,Int}, t_connections::Vector{Int}, rate_a::Vector{<:Real})
+    vr_to = [var(pm, nw, :vr, t_idx[1])[t] for t in t_connections]
+    vi_to = [var(pm, nw, :vi, t_idx[1])[t] for t in t_connections]
+    cr_to = var(pm, nw, :cr, t_idx)
+    ci_to = var(pm, nw, :ci, t_idx)
+
+    for idx in 1:length(rate_a)
+        if rate_a[idx]<Inf
+            psw_to_idx = JuMP.@NLexpression(pm.model,  vr_to[idx]*cr_to[idx] + vi_to[idx]*ci_to[idx])
+            qsw_to_idx = JuMP.@NLexpression(pm.model, -vr_to[idx]*ci_to[idx] + vi_to[idx]*cr_to[idx])
+
+            JuMP.@NLconstraint(pm.model, psw_to_idx^2 + qsw_to_idx^2 <= rate_a[idx]^2)
+        end
+    end
+end
+
+
+# BRANCH - Constraints - Quadratic
+
+"""
+	function constraint_mc_thermal_limit_from(
+		pm::AbstractQuadraticExplicitNeutralIVRModel,
+		nw::Int,
+		f_idx::Tuple{Int,Int,Int},
+		f_connections::Vector{Int},
+		rate_a::Vector{<:Real}
+	)
+
+For quadratic IVR models with explicit neutrals, 
+throw an error because this cannot be represented quadratically 
+without introducing explicit power variables.
+"""
+function constraint_mc_thermal_limit_from(pm::AbstractQuadraticExplicitNeutralIVRModel, nw::Int, f_idx::Tuple{Int,Int,Int}, f_connections::Vector{Int}, rate_a::Vector{<:Real})
+    if any(rate_a.<Inf)
+        error("""
+            A branch power bound cannot be represented quadratically in the default AbstractQuadraticExplicitNeutralIVRModel.
+            Either extend this quadratic formulation by including explicit branch power variables, or use AbstractNLExplicitNeutralIVRModel instead.""")
```

Comment:
Should this be an error? I think it might be better to just be a warning and to ignore the thermal limit, which is more in-line with current behavior elsewhere in the package